app/models/sitemap_indexer.rb

```diff
@@ -12,28 +14,30 @@ def initialize(sitemap_url:)
   end
 
   def index
-    sitemaps.any? ? enqueue_sitemaps : process_entries
+    enqueue_sitemaps
+    process_entries
```

Comment:
@ondrae, it's not the end of the world if these methods are called sequentially, but it's obscuring the fact that any given sitemap will *either* be a sitemap *or* a sitemap index, but never both. In keeping with the original logic, we could add a method to check to see if the sitemap is a sitemap index (as identified by the `<sitemapindex>` tag), in which case we would `enqueue_sitemaps`, otherwise `process_entries`.

Reply:
P.S. Maybe I'm overthinking this - we could also solve this with a comment...

Reply:
I went with 
```
  def index
    # A sitemap has either a sitemapindex or a urlset
    sitemapindex? ? enqueue_sitemaps : process_entries
  end

  private

  def sitemapindex?
    Saxerator.parser(sitemap).for_tag('sitemapindex').any?
  end
```

Reply:
👍